index.html

```diff
@@ -163,9 +73,11 @@ <h2 class="section-header">AeroGear is cross-platform</h2>
     <section class="whatsnew">
       <div class="container">
       
-        <h2 class="section-header">What is happening</h2>
+        <h2 class="section-header">Latest news</h2>
         
         <div class="row">
+        
+        <!-- 
```

Comment:
I believe we can remove these parts instead of commenting them out - we can always find them in the commit history.